src/main/java/com/google/sps/data/Card.java

```diff
@@ -107,7 +119,57 @@ public Entity createEntity() {
     card.setProperty("imageBlobKey", this.imageBlobKey);
     card.setProperty("rawText", this.rawText);
     card.setProperty("textTranslated", this.textTranslated);
+    card.setProperty("status", ClassStatus.ALIVE.toString());
```

Comment:
My example of ALIVE/DELETED/OTHER was just an example for a status with multiple states; in this case I think you should change status to "deleted" and use a boolean.

Also, you wouldn't need to turn it into a string; enums are integers and you should store the integers themselves.

Reply:
done